doc/api/v1/v1_bookmarks/check_for_a_non-existent_bookmark.json

```diff
@@ -23,14 +23,14 @@
   "requests": [
     {
       "request_method": "GET",
-      "request_path": "/api/v1/bookmarks/check?auth_token=person7%40example.com%3A1db5c50898dc7b5c5902e5a523ce61fdaabb5aaf3aa8e56a8886fdc013f298fd&url=http%3A%2F%2Fnot.found",
+      "request_path": "/api/v1/bookmarks/check?auth_token=person7%40example.com%3A4d9f54ab74ad33736b552ab0f38721ab349750b7d9abe7f83d57285f6875ef86&url=http%3A%2F%2Fnot.found",
```

Comment:
look at that beautiful churn

Reply:
yeah D: I've been looking and rspec_api_documentation has an "append only" feature for things like this I think? But once I get this into the docker build eventually, it shouldn't be an issue.